tests/unit/core/binarycodec/test_main.py

```diff
@@ -401,6 +402,32 @@ def test_claim(self):
         )
         self.assertEqual(encode_for_signing_claim(json), expected)
 
+    def test_batch(self):
+        flags = 1
+        transaction_ids = [
+            "ABE4871E9083DF66727045D49DEEDD3A6F166EB7F8D1E92FE868F02E76B2C5CA",
+            "795AAC88B59E95C3497609749127E69F12958BC016C600C770AEEB1474C840B4",
+        ]
+
+        json = {"flags": flags, "transaction_ids": transaction_ids}
+        actual = encode_for_signing_batch(json)
+        self.assertEqual(
+            actual,
+            "".join(
+                [
+                    # hash prefix
+                    "42434800",
+                    # flags
+                    "00000001",
+                    # transaction_ids length
+                    "00000002",
+                    # transaction_ids
+                    "ABE4871E9083DF66727045D49DEEDD3A6F166EB7F8D1E92FE868F02E76B2C5CA",
+                    "795AAC88B59E95C3497609749127E69F12958BC016C600C770AEEB1474C840B4",
```

Comment:
I like that this output is split according to different fields. It helps me understand better, nice work